integration/testUtils/index.ts

```diff
@@ -30,7 +30,7 @@ const createExpectPageObject = ({ page }: TestArgs) => {
     toBeHandshake: async (res: Response) => {
       // Travel the redirect chain until we find the handshake header
       // TODO: Loop through the redirects until we find a handshake header, or timeout trying
-      const redirect = await res.request().redirectedFrom().redirectedFrom().redirectedFrom().response();
+      const redirect = await res.request().redirectedFrom().redirectedFrom().response();
```

Comment:
I believe this change is necessary now that we're appropriately setting `x-forwarded-*` headers.